src/utils.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module enter/utils
+ */
+
+/**
+ * Returns iterator which has filtered attributes read from {@link module:engine/model/selection~Selection#getAttributes selection}.
```

Comment:
Should be: `Returns iterable...` as other docs for similar constructs.

Also might just skip `iterable blah blah` and just describe that "Returns attributes that should be preserved on the enter key. Filtering..."